app/components/move_message.py

```diff
@@ -31,9 +32,13 @@ async def select_channel(
         self, interaction: discord.Interaction, sel: discord.ui.ChannelSelect
     ) -> None:
         if self._used:
+            await interaction.response.send_message(
+                "Channel selector was already used.", ephemeral=True
+            )
```

Comment:
This should be reverted as #115 will handle this part of code + it's out of scope for this PR anyway.